packages/suite/src/components/firmware/OnboardingInitial.tsx

```diff
@@ -91,13 +101,11 @@ const BottomBar = () => {
     if (!device?.connected || !device?.features) return null;
```

Comment:
rm ?

Reply:
aha, you probably meant optional chaining operator :D. if used on !device?.connected, it is not needed on !device?.features. that should be right.